templates/cicd/buildspec.yml

```diff
@@ -1,61 +0,0 @@
-# Buildspec run in the build stage of your pipeline.
```

Comment:
We probably shouldn't delete this file :) The PR doesn't seem to need touching any of the files under the "templates" directory

Reply:
TY!